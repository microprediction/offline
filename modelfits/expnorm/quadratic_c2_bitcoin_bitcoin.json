{"g1": 0.10055088150097595, "g2": 0.3612801188158328, "logK": 0.9964012125246455, "loc": 0.11628289115419678, "logScale": -0.212942766733887}