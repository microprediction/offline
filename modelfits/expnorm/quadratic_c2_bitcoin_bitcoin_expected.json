{"g1": 0.5811814499832795, "g2": 9.551509458529427, "logK": 0.8763188313107307, "loc": -0.08716674262216238, "logScale": -1.80369636507726}